kumascript/macros/LiveSampleURL.ejs

```diff
@@ -8,18 +8,14 @@
 // $1   The URL from which to pull the sample; if not provided,
 //      the current page is assumed.
 
-var pageUrl = env.url;
-var base = pageUrl;
+var path = env.url;
 
 if ($1 && $1.length > 0) {
-    base = $1;
+    path = $1;
 }
 
-let url = new kuma.url.URL(base);
-wiki.ensureExistence(url.pathname);
-let liveSampleUrl = new kuma.url.URL(env.live_samples.base_url);
-url.host = liveSampleUrl.host;
-url.protocol = liveSampleUrl.protocol;
-url.pathname = url.pathname + '/_samples_/' + web.slugify(web.safeDecodeURIComponent($0));
+wiki.ensureExistence(path);
```

Comment:
I guess this is going to mean I'll have to fix conflicts in https://github.com/mdn/yari/pull/1180/files
Or, do you want to review that first and then come back to this?

Reply:
I'll review that one first, and then I'll come back to this.